### PR TITLE
[16.0][MIG] web_action_conditionable: Migration to 16.0

### PR DESCRIPTION
**Pull request by jasperjumelet**

### CODE REVIEW

**Review by chrisb-c01 — APPROVED** [review]

File: web_action_conditionable/README.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+::
+
+   <tree delete="false" create="false">
+
+with this module you can:
+
+::
+
+   <tree delete="state=='draft'" create="state!='sent'">
```
Comment (nguyenminhchien) [inline web_action_conditionable/README.rst:48]: @jasperjumelet Hello, this module doesn't work for me, have a look at the image below:
- i have add `delete="state=='draft'" create="state!='sale'"` to the tag `tree` of  Sales Order Lines: `<tree string="Sales Order Lines" editable="bottom" delete="state=='draft'" create="state!='sale'">`
- My test Order `S00006` is in state of `Sales Order` (sale). But the button `Add a product` still be there.
Screenshot from 2023-04-27 14-47-43

Comment (nguyenminhchien) [reply web_action_conditionable/README.rst:48]: btw, please make `runboat/build` work.

Comment (nguyenminhchien) [reply web_action_conditionable/README.rst:48]: Hello @jasperjumelet i have created this [pull request](https://github.com/jasperjumelet/web/pull/1) to make this module worked. Please review. 
Thanks,

Comment (StephaneMangin) [reply web_action_conditionable/README.rst:48]: @nguyenminhchien  Why not adding these changes https://github.com/jasperjumelet/web/pull/1 to the current PR ?

Comment (nilshamerlinck) [reply web_action_conditionable/README.rst:48]: Hello @StephaneMangin the new PR https://github.com/OCA/web/pull/2500 is the result of https://github.com/OCA/web/pull/2435 + https://github.com/jasperjumelet/web/pull/1 stashed in one single commit

**Comment by sebalix** [comment]
As there is no feedback from the author, we should close this PR in favor of https://github.com/OCA/web/pull/2500

**Pull request closed without merging**